### PR TITLE
[BPK-551] Add travis integration

### PR DESCRIPTION
**Pull request by k0nserv**
Adds initial travis CI. We'll do the docs deploy later

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+language: node_js
+node_js:
+  - "7.5.0"
+env:
+  - NPM_CONFIG_LOGLEVEL=warn
+install:
+  - npm install
+  - npm run bootstrap
```
Comment (tvooo) [inline .travis.yml]: Do we need this? `bootstrap` is run as postinstall script, meaning `npm install` should be sufficient. However, for some reason, Drone doesn't do it. Is that the same for Travis?

Comment (k0nserv) [inline .travis.yml]: Good shout I just copied it over from `.drone.yml`. Let's see what happens in the [build](https://travis-ci.org/Skyscanner/backpack/builds/256812600)

Comment (k0nserv) [inline .travis.yml]: You're right postinstall does work correctly and `npm run bootstrap` ends up running twice. I'll remove one entry

**Comment by k0nserv** [comment]
I added caching for `node_modules` hopefully that will just work even with Lerna creating symlinks

**Pull request merged**